src/API/src/db/migrations/1670412358270-news-lastUpdated.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class newsLastUpdated1670412358270 implements MigrationInterface {
+    name = 'newsLastUpdated1670412358270'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "news" ADD "lastUpdated" TIMESTAMP WITH TIME ZONE NOT NULL`);
```

Comment:
Will this work on existing news items? If it's not null, what does it set this to on existing rows?

Reply:
Decided that this is fine - there probably won't be any existing rows